Allow to follow our own versioning 'rcN', it's needed because fwbuilder
needs to find its files in ${LOCALBASE}/share/fwbuilder-PROJECT_VERSION.

Index: cmake/VERSION.cmake
--- cmake/VERSION.cmake.orig
+++ cmake/VERSION.cmake
@@ -3,7 +3,9 @@ set(FWBUILDER_XML_VERSION 24)
 set(PROJECT_VERSION_MAJOR "6")
 set(PROJECT_VERSION_MINOR "0")
 set(PROJECT_VERSION_PATCH "0")
-set(PROJECT_VERSION_EXTRA "-rc1")
+if (NOT DEFINED PROJECT_VERSION_EXTRA)
+	set(PROJECT_VERSION_EXTRA "-rc1")
+endif()
 set(PROJECT_GENERATION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}")
 set(PROJECT_GENERATION_SHORT "${PROJECT_VERSION_MAJOR}${PROJECT_VERSION_MINOR}")
 set(PROJECT_VERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}${PROJECT_VERSION_EXTRA}")
